app/views/sharks/edit.html.erb

```diff
@@ -2,5 +2,5 @@
 
 <%= render 'form', shark: @shark %>
 
-<%= link_to 'Show', @shark %> |
+<%= link_to 'Show', @shark, :class => "btn btn-primary btn-sm" %> <%= link_to 'Back', sharks_path, :class => "btn btn-primary btn-sm" %>
 <%= link_to 'Back', sharks_path %>
```

Comment:
There is an extra back button that was not removed, if it's not in use, it should be removed.